docs/api/zh/renderers/webgl/WebGLProgram.html

```diff
@@ -35,7 +35,7 @@ <h3>顶点着色器(无条件的):</h3>
 		uniform vec3 cameraPosition;
 		</code>
 		<code>
-		// default vertex attributes provided by Geometry and BufferGeometry
+		// default vertex attributes provided and BufferGeometry
```

Comment:
Typo: `by` changed to `and`

Reply:
283ce1ce24480a6b66d04286ccb8a49ad82d1a3e